opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -367,7 +367,18 @@ def create_span(
         parent: trace_api.ParentSpan = trace_api.Tracer.CURRENT_SPAN,
         kind: trace_api.SpanKind = trace_api.SpanKind.INTERNAL,
     ) -> "trace_api.Span":
-        """See `opentelemetry.trace.Tracer.create_span`.
+        """
+        Creating the span does not start it, and should not affect the tracer's
+        context. To start the span and update the tracer's context to make it
+        the currently active span, see :meth:`use_span`.
+
+        By default the current span will be used as parent, but an explicit
+        parent can also be specified, either a Span or a SpanContext.
+        If the specified value is `None`, the created span will be a root
+        span.
+
+        Applications that need to create spans detached from the tracer's
+        context should use this method.
```

Comment:
I would remove this paragraph, and maybe also rename the method to `_create_span`. Or we could commit to having this method be a "semi-public" interface of the SDK, to be overriden by any vendor-SDKs that need their own Span type but nevertheless want to reuse functionality from vendor SDKs.

In any case, *applications* (or any instrumentation use case really) must not use this method.